components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -204,6 +394,15 @@ public List<Property> getConfigurationProperties() {
         scope.setDisplayOrder(4);
         configProperties.add(scope);
 
+        Property googleOneTap = new Property();
+        googleOneTap.setName(GoogleOAuth2AuthenticationConstant.GOOGLE_ONE_TAP_ENABLED);
+        googleOneTap.setDisplayName("Enable One Tap");
```

Comment:
Shall we add this Display name as a constant like the `GoogleOAuth2AuthenticationConstant.GOOGLE_ONE_TAP_ENABLED`? WDYT?

Reply:
Fixed e8155c39133687b8e0f3dce714ead1fd77a8912f